cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -201,7 +288,7 @@ async def _batch_get_trustworthiness_score(
             capture_exceptions (bool): if should return None in place of the response for any errors or timeout processing some inputs
```

Comment:
if ``True``, the returned list will contain [TLMScore](#class-tlmscore) objects with error messages and retryability information in place of the score for any errors or timeout when processing a particular prompt from the batch.